pkg/geoip/geoip.go

```diff
@@ -18,6 +20,22 @@ type Location struct {
 
 // Fetch retrieves the location of the system calling this function
 func Fetch() (Location, error) {
+	geoipURLs := []string{"https://geoip.grid.tf/", "https://02.geoip.grid.tf/", "https://03.geoip.grid.tf/"}
```

Comment:
can this be defined in the global scope (outside the function body) not on the stack. it's more of a constant than a variable. like

```go
var (
   geoipURLs = []string{...}
)
```